microcosm_flask/fields.py

```diff
@@ -46,3 +49,48 @@ def _deserialize_by_name(self, value, attr, data):
             return getattr(self.enum, value)
         except AttributeError:
             self.fail('by_name', name=value)
+
+
+class TimestampField(Field):
+    """
+    Timestamp valued field, as either a unix timestamp (default) or isoformat string.
+
+    """
+    EPOCH = datetime(1970, 1, 1)
+
+    def __init__(self, use_isoformat=False, *args, **kwargs):
+        self.use_isoformat = use_isoformat
+        super(TimestampField, self).__init__(*args, **kwargs)
+
+    def _serialize(self, value, attr, obj):
+        """
+        Serialize value as a timestamp, either as a Unix timestamp (in float second) or a UTC isoformat string.
+
+        """
+        if value is None:
+            return None
+
+        if self.use_isoformat:
```

Comment:
The assumption here is that the internal value is always a Unix timestamp (in float seconds). We either return it verbatim (as a float) or encode it as an iso string.

It would be possible to generalize this to support datetimes internally, but I'd rather set the precedent that internal timestamps are numbers.